stats/internal/otel/prometheus/exporter.go

```diff
@@ -15,13 +15,15 @@
 //     see here: https://github.com/open-telemetry/opentelemetry-go/blob/v1.14.0/exporters/prometheus/exporter.go#L393
 //
 //  4. removed unnecessary otel_scope_info metric
+//
+//  5. added logic to remove and detect duplicated attributes (instead of concatenating them to stay compatible
+//     with gRPC)
```

Comment:
I would refrain from performing any further customisations to the original exporter. I would prefer extending our existing label deduplication logic in `stats` package instead, which now detects duplicate labels that the user provides, and include an extra validation for labels managed by our own library.

Reply:
Not a bad idea, although I don't think we'll go back to having the original exporter. It would require too many changes to our dashboards, plus the exporter is fairly small/simple, it shouldn't be much trouble to maintain it.

Anyhow, I moved the logic into `stats` now, the only concern is the duplication of sanitization logic. Have a look let me know what you think :+1: